index.js

```diff
@@ -9,7 +9,7 @@ module.exports.pitch = function(remainingRequest) {
 	return "require(" + JSON.stringify("!!" + path.join(__dirname, "addScript.js")) + ")"+
 			"(require(" +
 			JSON.stringify("!!" + require.resolve("raw-loader") + "!" + remainingRequest) + ")" +
-				(this.debug ?
+				(this.debug || (this.query && this.query.sourceMap) ?
```

Comment:
@see https://github.com/webpack-contrib/script-loader/pull/31/files#diff-168726dbe96b3ce427e7fedce31bb0bc for a potential better way to do it